### PR TITLE
Add Discord Support to Creat Circle Form

### LINKED ISSUE #601: User Support Circle Creation Form Updates
### **Summary**
We want to update the circle creation page to aid with user support. 
This request includes adding a Header, changing the Circle Point of Contact field, and adding the Discord support field.

[Proposal / Rationale](https://www.notion.so/coordinape/New-User-Support-Quick-Hacks-6f8f647147ee472c8d1c47ef931a2f91?d=a5b8d20b9cfc42c8b886c92fda9bd019)

### **Specification**

Design: https://www.figma.com/file/KciBaNjP5Q9idchsxMp9ek?embed_host=notion&kind=&node-id=190%3A790&viewer=1

Add Header "Coordinape Support"

**Circle Point of Contact Field** 
- Add a subheader under Circle Point of Contact with the following copy "We use this for follow-up & support."
- Change Helper Text with the text field to "Discord #0000, Telegram, Twitter or Email."
- Make this field required before the user moves forward
- Error Message Text  "Circle Point of Contact is Required"

**New Need More Help? Field**
- Add a new field to the right of Circle Point of Contact 
- Field Header: Need More Help?
- Subheader: Join Our Discord for Information & Support
- Add Discord Button with Call to Action Text "Join Coordinape Discord"  using this link "https://discord.gg/BG3fDAvzuB"

[You Can find CSS & Visual Styling in the Figma Inspect Section.](https://www.figma.com/file/KciBaNjP5Q9idchsxMp9ek/Coordinape-Tool-Tip-Additions?node-id=190%3A790)

### **Acceptance Criteria** 
**Scenario: User forgets to submit Point of Contact**
- Given: The user navigates to Create a Circle
- When: The Circle Point of Contact field is empty
- Then: The system remains on the same page
- Then: The displays error message "Circle Point of Contact is Required."

**Scenario: User needs support** 
- Given: The user navigates to Create a Circle
- When: The user selects Join Coordinape Discord
- Then: The system opens a new tab with a Discord Invite Link 

**Stakeholders**
@spydermonkee @ivyquinzel

### PR DESCRIPTION
**Pull request by amrro**
fixes #601 

This adds the Coordinape Discord support to the Create Circle and follow up the new design described in the #601.

### CODE REVIEW

File: src/components/ApeTextField/ApeTextField.tsx
Comment (crabsinger) [inline src/components/ApeTextField/ApeTextField.tsx:218]: seems we could remove this?

**Review by levity — APPROVED** [review]
Code looks fine to me; just waiting for sign-off from the design team at #601

**Comment by crabsinger** [comment]
@amrro the helper text for "Circle Point of Contact" doesn't match the spec.

What you have in the PR:
Screen Shot 2022-03-14 at 10 15 15 AM

What the spec says:

Screen Shot 2022-03-14 at 10 15 22 AM

Can you update the text to "Discord #0000, Telegram, Twitter or Email"

File: src/pages/CreateCirclePage/CreateCirclePage.tsx
Comment (crabsinger) [inline src/pages/CreateCirclePage/CreateCirclePage.tsx:223]:
```suggestion
                  placeholder="Discord #0000, Telegram, Twitter or Email "
```

**Comment by crabsinger** [comment]
@amrro nvm, I've gone ahead and pushed this update to the branch directly.

**Review by crabsinger — APPROVED** [review]

**Pull request merged**
